core-web/libs/ui/src/lib/modules/dot-dialog/dot-dialog.component.scss

```diff
@@ -14,7 +14,7 @@
         position: absolute;
         right: 0;
         top: 0;
-        z-index: 1000;
+        z-index: 1103;
```

Comment:
mmm maybe this change break a confirm dialogs component
Have you tried (if is needed)  to open a confirmation dialog on this component?

Reply:
We need to test this when we fix this https://github.com/dotCMS/core/issues/28177